packages/generator/src/artifacts/css/flat-css.ts

```diff
@@ -10,7 +10,9 @@ export const generateFlattenedCss = (ctx: Context) => (options: { files: string[
   const { theme: { keyframes } = {}, preflight, minify, staticCss } = ctx.config
 
   const unresolved = [
-    '@layer reset, base, tokens, recipes, utilities;',
+    `@layer ${Object.values(ctx.layers)
+      .map((l, i, arr) => (i === arr.length - 1 ? l : `${l}, `))
+      .join('')};`,
```

Comment:
I'm not sure about this, when running this code:

```ts
const layers = {  base: "base", tokens: "tokens", recipes: "recipes", reset: "reset", utilities: "utilities", other: "other" }

Object.values(layers).map((l, i, arr) => (i === arr.length - 1 ? l : `${l}, `))
      .join('')
```

I get this result, which is in the wrong order : `'base, tokens, recipes, reset, utilities, other'`

---
something like this looks a bit better

```ts
const layers = {  base: "base_suffixed", tokens: "tokens", recipes: "recipes", reset: "reset", utilities: "panda_utilities", other: "other" }

const layersStr = '@layer reset, base, tokens, recipes, utilities;'
const updated = Object.keys(layers).reduce((acc, name) => acc.replace(name, layers[name]), layersStr)

console.log(updated)
// `@layer reset, base_suffixed, tokens, recipes, panda_utilities;`
```